### ISSUE TITLE
installation documentation

### ISSUE THREAD

**Issue by hodiapa** [PagerDuty/Nut#26]
How to install this utility on Ubuntu/unix?
Being not familiar with golang, how does one go about installing the NUT package to use for LXC container creation?

**Comment by theckman** [comment]
It looks like the latest tag has a binary attached to it to be downloaded:

* https://github.com/PagerDuty/Nut/releases/tag/0.2

If you want to build your own newer version from source, you'll need to follow this guide to get Go installed on your workstation:

* https://golang.org/doc/install

Once you do that you shoud be able to compile Go binaries. Looking at the `Makefile` included in this project, it seems like `make bin` should build you a binary:

* https://github.com/PagerDuty/Nut/blob/82df407d329ea0dbf142cd357e3974144243d8ad/Makefile

**Comment by hodiapa** [comment]
Thanks will give it a try. Would be nice to include this info into the readme

**Comment by theckman** [comment]
Agreed. I think there is often an assumption, in Go project README files, that getting your workstation ready to compile Go programs is a bit beyond the scope.

Although, thinking about it more I guess it proliferates beyond Go even in to other compiled languages like C/C++. It's assumed that your workstation is ready to build that source code.

I'm not directly involved with this project, so my opinions may not have any weight, but I'd guess that they would appreciate a PR to amend the `README.md` file with some build instructions.

**Comment by hodiapa** [comment]
Is this project still active? Doesn't seem like much activity here.. wonder if it is feature complete...

**Comment by theckman** [comment]
I'm not sure it's feature complete, but I think the needs of the authors may have changed (for example: maybe using Docker directly instead of LXC). That's just based on some of the issues that are still not addressed. Some of them seem like they'd be needed if this were being used at a scale beyond experimentation.
